content/_data/changelogs/weekly.yml

```diff
@@ -1654,6 +1654,70 @@
           title: security advisory
         - url: /blog/2017/12/14/security-update/
           title: announcement blog post
+- version: "2.96"
+  date: 2017-12-17
+  changes:
+    - type: major bug
+      pull: 3193
+      issue: 48365
+      message: >
+        Make sure detached plugins (plugins whose functionality used to be part of Jenkins itself) are installed when upgrading Jenkins past the version at which the plugin was detached.
+
+    - type: rfe
+      pull: 3129
+      issue: 22474
+      message: >
+        Do not require CSRF crumb to be provided when the request is authenticated using API token.
+    - type: rfe
+      issue: 48405
+      pull: 3169
+      message: >
+        Improve robustness and error handling when trying to delete files or making them writable by switching to NIO.
+    - type: rfe
+      issue: 47324
+      pull: 3173
+      message: >
+        Improve robustness and error handling when trying to copy or move files or creating directories by switching to NIO.
+    - type: rfe
+      pull: 3176
+      message: >
+        Improve Chinese translation.
+
+    - type: bug
+      pull: 3187
+      message: >
+        Update Stapler from 1.253 to 1.254 to make the form that shows up when a URL requiring <code>POST</code> is accessed using a different HTTP verb work with CSRF protection enabled.
+      references:
+        - issue: 34254
+        - url: https://github.com/stapler/stapler/blob/master/CHANGELOG.md#1254
+          title: Stapler changelog
+    - type: bug
+      pull: 3198
+      issue: 48505
+      message: >
+        Fix a performance regression in Jenkins 2.86 due to lock contention in <code>ExtensionList</code>.
+    - type: bug
+      pull: 3175
+      issue: 48383
+      message: >
+        Trigger <code>SecurityListener#loggedIn</code> events on programmatic login during self-registration when using <code>HudsonPrivateSecurityRealm</code>.
+
+    - type: rfe
+      pull: 3074
+      issue: 27027
+      message: >
+        Developer: Capture more authentication-related events.
```

Comment:
"... in SecurityListener"?
Also it makes sense to specify a link to https://jenkins.io/doc/developer/extensions/jenkins-core/#securitylistener